features/machine/machine_misc.feature

```diff
@@ -106,3 +106,27 @@ Feature: Machine misc features testing
       | o             | json                         |
     Then the expression should be true>  @result[:parsed]['webhooks'][0]['clientConfig']['service']['port'] == 443
     """
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-37744
+  @admin
+  Scenario Outline: kube-rbac-proxy should not expose tokens, have excessive verbosity
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Then I use the "<project>" project
+
+    Given a pod becomes ready with labels:
+      | api=clusterapi| <labels>  |
```

Comment:
`| api=clusterapi | <labels> |`

Reply:
Thanks a lot @sunzhaohua2 , added those , it seems both are bug 
  [07:42:49] INFO> Exit Status: 0
      | resource_name | <%= pod.name %> |
      | c             | kube-rbac-proxy |
        Then the output should not contain:                                                       # features/step_definitions/common.rb:33
      | Response Headers |
      pattern found: Response Headers (RuntimeError)
will follow up on those ..